rootfs/etc/profile.d/_50-workspace.sh

```
@@ -0,0 +1,28 @@
+	for path in "${GEODESIC_HOST_PATHS[@]}"; do
+		if [[ "$path" == "${dir}/" || "$dir" == "$path"* ]]; then
+			return 0
+		fi
+	done
+	return 1
+}
+
+if [[ $SHLVL == 1 ]]; then
+	if [[ -d ${WORKSPACE_FOLDER:=${WORKSPACE_MOUNT}} ]]; then
+		green "# Initial working directory configured as ${WORKSPACE_FOLDER}"
+		cd "${WORKSPACE_FOLDER}"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)